policyengine_us/variables/gov/states/mt/tax/income/credits/ctc/mt_ctc.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class mt_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MT CTC"
+    definition_period = YEAR
+    unit = USD
+    documentation = "Montana Child Tax Credit"
+    reference = "https://leg.mt.gov/bills/2023/billpdf/HB0268.pdf"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.credits.ctc
+        # income limit
+        gross_income = tax_unit("adjusted_gross_income", period)
+        income_eligible = gross_income <= p.income_threshold
+        investment_income_eligible = (
+            tax_unit("net_investment_income", period) <= p.investment_threshold
+        )
+        person = tax_unit.members
+        dependent = person("is_tax_unit_dependent", period)
+        meets_age_limit = person("age", period) < p.child_age_eligibility
+        eligible_child = dependent & meets_age_limit
+
+        eligible_children = tax_unit.any(eligible_child)
```

Comment:
Do you mean `tax_unit.sum`?

Please add a test case with multiple eligible children to check.

If it is `tax_unit.any` (that is, if it's not scaled by number of kids), then please rename to `has_eligible_children`.